src/templates/type/moment-select.html

```diff
@@ -7,7 +7,7 @@
                         class="select"
                         ng-model="hours"
                         ng-disabled="disable"
-                        ng-options="hour as hour for hour in hourChoices">
+                        ng-options="hour as (trailingHours + hour).slice(-(trailingHours.length)) for hour in hourChoices">
```

Comment:
Filter would be great. You can use it in `ng-options` and `ng-repeat` by wrapping with `()`:
```html
<select ng-options="hour as (trailHours ? (hour|trailingZeros) : hour) for hour in hourChoices"></select>
```